assets/interfaces/common.ts

```diff
@@ -84,8 +84,8 @@ export interface IOccurStatus {
 export interface IDateFilters{
     name: string,
     default: boolean,
-    query?: Array<{}>
-    filter: Array<['today', 'last_week', 'last_30_days', '']>
+    query?: string,
```

Comment:
I think it's not used on the client right? so you might as well remove it

Reply:
yep